Assets/Scripts/Audio/Sound.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+//This class is here for the devs to edit or add settings to any of the other sound scripts that are available
+[System.Serializable]
+public class Sound
+{
+	public string name; 
```

Comment:
quick addendum to my comment earlier about [SerializeField] vs public. i do think it's fine to use public in the case of "plain old data" classes like this. so i think this case is completely fine